### PR TITLE
Release

### PR DESCRIPTION
**Pull request by Samuel2507**

### CODE REVIEW

**Comment by Pezzawinkle** [comment]
may want to refresh this before bumping out

File: omnilib/data-final-fixes.lua [hunk 1]
```diff
@@ -1,9 +1,6 @@
-for _,pack in pairs({"automation-science-pack","logistic-science-pack","chemical-science-pack","production-science-pack","military-science-pack","utility-science-pack"}) do
-	data.raw.tool[pack].icon = nil
-	data.raw.tool[pack].icons = {{icon="__omnilib__/graphics/icons/science-pack/"..pack..".png",icon_size=64}}
-	if data.raw.technology[pack] then
-		data.raw.technology[pack].icon = "__omnilib__/graphics/technology/"..pack..".png"
-	end
+--check setting has actually been forced off
+if mods["bobtech"] then
+  settings.startup["bobmods-tech-colorupdate"].value = false
 end
```
Comment (SqFKYo) [inline omnilib/data-final-fixes.lua:4]: Why this check is both in final-fixes and data?

File: omnimatter_compression/control.lua [hunk 2]
```diff
@@ -49,8 +49,10 @@ end)
 
 script.on_event(defines.events.on_research_finished, function(event)
   local tech = event.research
+  local level = tech.level
   --local name = string.match(tech.name,"(.*)%-%d*")
```
Comment (SqFKYo) [inline omnimatter_compression/control.lua:53]: Why add alias if you're using still tech.level below?

File: omnimatter_energy/info.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
     "(?)bobplates",
     "?boblogistics",
     "(?)deadlock-integrations",
-    "?bobassembly"
+    "?bobassembly",
+    "!aai-industry"
 ]
```
Comment (SqFKYo) [inline omnimatter_energy/info.json:18]: Does info.json support trailing commas on the last line? Would make diffs easier to read.

File: omnimatter_compression/prototypes/compress-items.lua [hunk 4]
```diff
@@ -294,11 +295,25 @@ for _, group in pairs({"item", "ammo", "module", "rail-planner", "repair-tool",
...
+    end
 	end
 end
+--log(serpent.block(excluded_items))
+
+for _, rec in pairs(data.raw.recipe) do
+  for _, dif in pairs({"normal", "expensive"}) do
+    for _, ing in pairs(rec[dif].ingredients) do
+      if excluded_items[ing.name] then
+        log("Excluded recipe '"..rec.name.."' due to '"..ing.name.."' beeing on the blacklist")
```
Comment (SqFKYo) [inline omnimatter_compression/prototypes/compress-items.lua:309]: Typo (beeing vs being)

File: omnimatter_energy/info.json (hunk 3 above)
Comment (Samuel2507) [reply omnimatter_energy/info.json:18]: Nope, besides the VS errors factorio does not load that mod at all if you add a comma there

**Comment by SqFKYo** [comment]
Will add missing changelog for science to master after merge.

**Pull request merged**